grails-app/assets/stylesheets/record-view.css

```diff
@@ -20,6 +25,13 @@ td.noStripe {
 
 .occurrenceTable td {
     word-wrap: break-word;
+    border-left: 0;
+    border-right: 0;
+}
+
+.data-quality-table td {
```

Comment:
Have you tried removing ``table-bordered`` class?

Reply:
Yes but it also removed the outer vertical borders which I want to keep